resources/test-runner.mjs

```diff
@@ -24,7 +24,7 @@ export class TestRunner {
         const testName = this.#test.name;
         const syncStartLabel = `${suiteName}.${testName}-start`;
         const syncEndLabel = `${suiteName}.${testName}-sync-end`;
-        const asyncStartLabel = `${suiteName}.${testName}-async-start`;
+        const asyncStartLabel = syncEndLabel;
```

Comment:
I'd rather remove the name and use `syncEndLabel` directly below. I think this makes it more obvious what the PerformanceTiming marker will be about.

Reply:
done.